drivers/gpu/drm/i915/gem/i915_gem_execbuffer.c

```diff
@@ -3368,11 +3371,18 @@ i915_gem_do_execbuffer(struct drm_device *dev,
 	struct sync_file *out_fence = NULL;
 	int out_fence_fd = -1;
 	int err;
+    u64 start_time = ktime_get_raw_ns();
 
 	BUILD_BUG_ON(__EXEC_INTERNAL_FLAGS & ~__I915_EXEC_ILLEGAL_FLAGS);
 	BUILD_BUG_ON(__EXEC_OBJECT_INTERNAL_FLAGS &
 		     ~__EXEC_OBJECT_UNKNOWN_FLAGS);
 
+    const struct cred* cred = get_current_cred();
+    const unsigned int uid = cred->euid.val;
+    put_cred(cred);
+    trace_gpu_work_period(i915->drm.primary->index, uid,
```

Comment:
hard code the gpu time, need add more description for this workaround

Reply:
Done